src/Components/TabItem/TabItem.tsx

```diff
@@ -0,0 +1,24 @@
+import { FC } from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
+
+type Props = {
+  tab: Tab
+};
+
+export const TabItem: FC<Props> = ({ tab }) => {
+  const { tabId } = useParams();
```

Comment:
You've already used the `useParams` in a parent component, would be better provide it via props